daprdocs/content/en/reference/environment/_index.md

```diff
@@ -8,12 +8,9 @@ weight: 300
 
 The following table lists the environment variables used by the Dapr runtime, CLI, or from within your application:
 
-| Environment Variable             | Used By          | Description |
-|----------------------------------|------------------|-------------|
-| DAPR_HTTP_PORT                   | Your application | The HTTP port that Dapr is listening on. Your application should use this variable to connect to Dapr instead of hardcoding the port value. Injected by the `dapr-sidecar-injector` into all the containers in the pod.
-| DAPR_GRPC_PORT                   | Your application | The gRPC port that Dapr is listening on. Your application should use this variable to connect to Dapr instead of hardcoding the port value. Injected by the `dapr-sidecar-injector` into all the containers in the pod.
-| DAPR_TOKEN_API                   | Your application | The token used for Dapr API authentication for requests from the application. Read [enable API token authentication in Dapr]({{< ref api-token >}}) for more information.
-| APP_TOKEN_API                    | Your application | The token used by the app to authenticate requests from Dapr. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information.
```

Comment:
This should be kept. The name is wrong, it should be APP_API_TOKEN

Reply:
@msfussell  is there any conflict for APP_API_TOKEN doc?

- 2nd column (Used by): `Dapr sidecar`
- 3nd column: `The token used by the application`

Reply:
APP_API_TOKEN - Yeah, its kind of confusing. I think what you have Dapr sidecar is the best clarity, since the env variable is used "read" by the Dapr sidecar and then handed to the application on each request. Now the application also has to have this TOKEN to check the calls.

Reply:
Yes. I agree with you. Please make this change.

Reply:
Fixed